### PR TITLE
amigo remaining services

### PR DESCRIPTION
**Pull request by akash1810**

### CODE REVIEW

**Comment by akash1810** [comment]
tested on TEST, deploying to PROD

File: .editorconfig [hunk 1]
```diff
@@ -14,3 +14,6 @@ indent_size = 2
 
 [*.yml]
 indent_size = 2
+
+[*.yaml]
```
Comment (gtrufitt) [inline .editorconfig]: No biggie, for future reference this could be `*.{yml,yaml}`  above

Comment (akash1810) [reply .editorconfig]: https://github.com/guardian/grid/pull/2094/files 😄

Comment (gtrufitt) [reply .editorconfig]: 💯

**Pull request merged**